frontend/src/main/java/com/google/tripmeout/frontend/storage/TripStorage.java

```diff
@@ -0,0 +1,71 @@
+package com.google.tripmeout.frontend.storage;
+
+import com.google.tripmeout.frontend.TripModel;
+import java.util.List;
+
+/**
+ * stores TripModel objects
+ */
+public interface TripStorage {
+  /**
+   * adds a TripModel object to storage
+   *
+   * @param trip the TripModel object to add to storage
+   *
+   * @throws a TripAlreadyExists exception if there is a TripModel object 
+   *     already in storage with the same tripId as the trip
+   */
+  public void addTrip(TripModel trip);
+
+  /**
+   * removes from storage the TripModel object whose tripId matches the given tripId
+   *
+   * @param tripId the id to match TripModel object's tripId field on
+   *
+   * @throws a TripModelNotFound exception if there is no TripModel object with 
```

Comment:
Should this exception just be `TripNotFound` for continuity with the places exceptions?